minvws/nl-kat-coordination
# Fix filtering on plugin_id for normalizers

### Changes

Updated filtering on the queryparam `plugin_id` for normalizers. The correct `"and"` clause wasn't present. And this has been added now.

### Issue link

No issue link

### QA notes

1. On `main`, go through the onboarding, create the organisation `dev` and create a scan for a domain
2. On the command line issue the following:

```bash
$ curl 'http://localhost:8004/tasks?scheduler_id=normalizer-dev&task_type=normalizer&plugin_id=kat_dnssec_normalize'
```

3. This should relay the following message:

```javascript
{
  "detail": "error occurred while accessing the database [exception: An error occurred: 'str' object has no attribute 'field']"
}
```

4. Checkout this branch and follow step 1 and 2 again

5. No error should be relayed

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
